### ISSUE TITLE
Infinite loading

### ISSUE THREAD

**Issue by AsuToki360** [judy-n/WhatsAppWrapped#1]
As the title says, it's on an infinte "loading" when I upload the chat file...

![image](https://user-images.githubusercontent.com/40326499/147505551-ec6e3d79-02cd-4286-876e-a267872d6aff.png)

**Comment by judy-n** [comment]
@MProductionYT We fixed an issue with Android exports, try again and lemme know if it works now!

**Comment by AsuToki360** [comment]
> @MProductionYT We fixed an issue with Android exports, try again and lemme know if it works now!

Well, it worked this time, but now it shows this for some reason: 
![image](https://user-images.githubusercontent.com/40326499/147600338-420e7e55-deda-486d-a85c-8fe2cd9aa632.png)

**Comment by judy-n** [comment]
@MProductionYT Hmm could you show me an example of what the format of your chat looks like? Cause so far we only tested it on iOS and 2 kinds of Android outputs. Thanks!

**Comment by AsuToki360** [comment]
> @MProductionYT Hmm could you show me an example of what the format of your chat looks like? Cause so far we only tested it on iOS and 2 kinds of Android outputs. Thanks!

Sure, but where can I check the format of the chat? (Also my English is kinda bad, so I apologize for that)

**Comment by judy-n** [comment]
@MProductionYT Just open the txt file that you're trying to upload and show me what a line looks like, for example: 

`[2017-09-07, 7:58:30 AM] Judy: Hello`

**Comment by AsuToki360** [comment]
> @MProductionYT Just open the txt file that you're trying to upload and show me what a line looks like, for example:
> 
> `[2017-09-07, 7:58:30 AM] Judy: Hello`

Mine shows up like this: 
![image](https://user-images.githubusercontent.com/40326499/147608266-e12e98f0-2ea5-405a-a367-877a47e5401a.png)

**Comment by judy-n** [comment]
@MProductionYT Ok I just pushed a possible fix, could you try again?

**Comment by AsuToki360** [comment]
> @MProductionYT Ok I just pushed a possible fix, could you try again?

Well, I tried again and the same thing happened... 
![image](https://user-images.githubusercontent.com/40326499/147614585-cfd8e620-18e3-4fdc-8022-d24837e72aa3.png)

**Comment by DarkHollow** [comment]
My .txt lines are like this:

`2/9/21 2:40 - Roberto: 🥲`

but I get error too:

![image](https://user-images.githubusercontent.com/1090925/147767946-1d7063a8-37c3-4fb4-b741-97597a9938fe.png)

it seems that there are different export formats on Android

**Comment by judy-n** [comment]
@DarkHollow Hi, thanks for showing me this! I just pushed a fix, could you try again?

**Comment by AsuToki360** [comment]
> @DarkHollow Hi, thanks for showing me this! I just pushed a fix, could you try again?

Well, now it works, but some stuff are glitched out, specially the names: 
![image](https://user-images.githubusercontent.com/40326499/147769362-f634d2c7-803e-43da-9a15-79980e701b3b.png)
![image](https://user-images.githubusercontent.com/40326499/147769422-a68f4653-841d-43f4-ae7c-43623f097274.png)

Both names show up as "m." for some reason.

**Comment by DarkHollow** [comment]
> @DarkHollow Hi, thanks for showing me this! I just pushed a fix, could you try again?

Now for me it's working fine, the only problem is the texting streak (0 days, but it was all days)

**Comment by AsuToki360** [comment]
> > @DarkHollow Hi, thanks for showing me this! I just pushed a fix, could you try again?
> 
> Now for me it's working fine, the only problem is the texting streak (0 days, but it was all days)

@DarkHollow Yeah, that's what I am getting as well on the pictures that I showed up recently

**Comment by Gigi780075** [comment]
@judy-n the link doesn't open
